mypy/modulefinder.py

```diff
@@ -70,38 +70,60 @@ def __init__(self,
                  options: Optional[Options] = None) -> None:
         self.search_paths = search_paths
         self.fscache = fscache or FileSystemCache()
-        # Cache find_lib_path_dirs: (dir_chain, search_paths) -> list(package_dirs, should_verify)
-        self.dirs = {}  # type: Dict[Tuple[str, Tuple[str, ...]], PackageDirs]
+        # Cache for get_toplevel_possibilities: dir_chain -> (id -> list(package_dirs))
```

Comment:
Nit: Maybe document that `dir_chain` contains module search directories. Maybe also explain that `id` is a top-level package name.

Reply:
oh oops, `dir_chain` should actually be `search_path`